tests/dbus/test_dbus_client.cpp

```diff
@@ -54,6 +54,11 @@ using otbr::DBus::LinkModeConfig;
 using otbr::DBus::OnMeshPrefix;
 using otbr::DBus::SrpServerInfo;
 using otbr::DBus::ThreadApiDBus;
+
+#if OTBR_ENABLE_TREL
+using otbr::DBus::TrelInfo;
```

Comment:
nit: Probably we can put this in `CheckTrelInfo` so there'll be less fragmentation.

Reply:
done